Dockerfile

```diff
@@ -99,11 +104,37 @@ COPY benchmarks/holoscan_flow_benchmarking/requirements.txt /tmp/benchmarking_re
 RUN pip install -r /tmp/benchmarking_requirements.txt
 ENV PYTHONPATH=/workspace/holohub/benchmarks/holoscan_flow_benchmarking
 
-# For RTI Connext DDS
+# --------------------------------------------------------------------------
+#
+# Set up common packages for developing with Yuan Qcap
+#
+# --------------------------------------------------------------------------
+FROM dev as yuan-qcap
+
+# Qcap dependency
+RUN apt update \
+    && apt install --no-install-recommends -y \
+        libgstreamer1.0-0 \
+        libgstreamer-plugins-base1.0-0 \
+        libgles2 \
+        libopengl0
+
```

Comment:
Does this really need to be in the Holohub default image? Can't we make specific Dockerfile for those operators?

Reply:
Probably not, and other components such as DDS may also be removable. I'd rather keep those decisions out of the scope of this PR, would it be OK if we follow up on those decisions in a later change?